src/pages/Confirm/index.jsx

```diff
@@ -42,13 +51,20 @@ const Confirm = () => {
   /** @param {React.FormEvent<HTMLFormElement>} event */
   const submitConfirmation = async event => {
     // Submit form here
-    if (event.name && event.RUT && event.phone) {
-      confirmPackage(event, packageId)
+    // @ts-ignore
+    if (event.RUT) {
+      // @ts-ignore
+
+      confirmPackage(event, id)
     } else {
       alert('Por favor complete los campos')
     }
   }
```

Comment:
Why here is changed the condition?

Reply:
Yes, here we are using only the RUT to confirm the package